addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRuleUnitTest.java

```diff
@@ -93,287 +101,640 @@ void shouldReturnExpectedMappings() {
                 is(equalTo(CommonAlertTag.WSTG_V42_INPV_05_SQLI.getValue())));
     }
 
-    @Test
-    void shouldTargetDbTech() {
```

Comment:
It would have been better if this was done in a separate commit.

Reply:
Good point. I'll keep code reorganization separate from code changes going forward.

Reply:
Updated to remove those changes